src/gax/src/http_client/mod.rs

```diff
@@ -30,7 +31,9 @@ impl ReqwestClient {
         let cred = if let Some(c) = config.cred {
             c
         } else {
-            ClientConfig::default_credential().await?
+            create_access_token_credential()
+                .await
+                .map_err(Error::authentication)?
```

Comment:
For a future PR.  the `gax::Error` class should (maybe) only consume `CredentialsError` in `Error::authentication`.

Reply:
Ack